tutor-multi-chart/Chart.yaml

```diff
@@ -20,3 +20,11 @@ dependencies:
 - name: traefik
   version: "20.3.0"
   repository: https://traefik.github.io/charts
+- name: metrics-server
+  version: "3.8.3"
+  repository: https://kubernetes-sigs.github.io/metrics-server/
+  alias: metricsserver
+- name: vertical-pod-autoscaler
+  version: "6.0.3"
+  repository: https://cowboysysop.github.io/charts/
+  alias: vpa
```

Comment:
Metrics server totally makes sense, though we do need to make sure we document a variable that can be used to not try to install it, for people who already have it installed on their cluster.

For VPA, how will we use it? See my questions/update on https://github.com/openedx/tutor-contrib-multi/issues/2#issuecomment-1423217262

Reply:
@bradenmacdonald I added a couple of conditions to define is the sub-charts are going to be installed or not.
Regarding the VPA topic, my approach is still naive for now. In the plugin, I created there are variables that enabled VPA for the main deployments (LMS, CMS, and workers) and creates VPA objects that will increase/decrease the allocated resources of every pod according to current cluster usage

Reply:
More details here: https://www.kubecost.com/kubernetes-autoscaling/kubernetes-vpa/